src/cap-description.adoc

```diff
@@ -613,14 +613,21 @@ bounds are formed of two or three sections:
 * Lower bits are set to zero
 ** This is only if there is an internal exponent (EF=0)
 
-.Composition of address bounds
-[#comp_addr_bounds,options=header,align="center"]
+.Composition of the decoded top address bound
+[#comp_addr_bounds,options=header,align="center",cols="2,4,2,2"]
 |==============================================================================
-| Configuration  | Upper section | Middle Section | Lower section
-| EF=0           | address[MXLEN-1:E + MW] + ct | T[MW - 1:0] | {E{1'b0}}
-| EF=1, i.e. E=0 | address[MXLEN-1:MW] + ct   2+| T[MW - 1:0]
+| Configuration  | Upper section (if E + MW &#8804; MXLEN) | Middle Section | Lower section
```

Comment:
If E+MW > MXLEN then is the top sectioin excluded completely? I think that makes sense (maybe worth clarifying in the text?)

Reply:
Exactly. The bottom section is E bits wide, the middle section is MW bits wide, and the top section picks up the slack, so has size MXLEN + 1 - MW - E. If MW + E > MXLEN then there's no slack for it to take up. That only happens for the very largest exponents. I'll clarify in the text.